index.html

```diff
@@ -1787,6 +1935,296 @@ <h2>
+          mapping [=MIME types=] to a list of extensions. Extensions have to be
+          strings that start with a "." and only contain [=valid suffix code
+          points=]. Additionally extensions are limited to a length of 16 code
+          points.
+        </p>
+        <p>
+          In addition to complete [=MIME types=], "*" can be used as the
+          subtype of a MIME type to match, for example, all image formats with
+          "image/*". However, the top-level type MUST be defined in [[rfc2046]]
+          and listed in [[IANA_MEDIA_TYPES]].
+        </p>
+        <p>
+          Websites MUST always provide both [=MIME types=] and file extensions
+          for each [=file handler/accept=] entry. This format is required by
+          operating systems that use [=MIME types=] to specify the [=file
+          types=]. On platforms that only use extensions, these can easily be
+          used instead.
+        </p>
+      </section>
+      <section>
+        <h3>
+          `icons` member
+        </h3>
+        <p>
+          The <a>file handler</a>'s <code><dfn data-dfn-for=
+          "file handler">icons</dfn></code> member lists images that serve as
+          graphical representations of a [=file type=] on a platform. This MAY
+          be replaced by the user agent for privacy & security reasons.
+        </p>
+      </section>
+      <section>
+        <h2>
+          Processing file handler items
+        </h2>
+        <p>
+          To <dfn>process a file handler item</dfn>, given [=ordered map=]
+          |item:ordered map|, [=URL=] |start_url:URL|, [=URL=] |scope:URL|, and
+          [=URL=] |manifest URL:URL|.
+        </p>
+        <ol class="algorithm">
+          <li>Return failure if it's the case that:
+            <ul>
+              <li>|item| is not [=ordered map=].
+              </li>
+              <li>|item|["action"] doesn't [=map/exist=] or is not a
+              [=string=].
+              </li>
+              <li>|item|["name"] [=map/exists=] and is not a [=string=].
+              </li>
+              <li>|item|["accept"] doesn't [=map/exist=].
+              </li>
+              <li>|item|["accept"] is not a [=dictionary=].
+              </li>
+              <li>|item|["accept"] is [=list/empty].
+              </li>
+            </ul>
+          </li>
+          <li>Let |url:URL| be the result of [=URL Parser|parsing=]
+          |item|["action"] with |start_url URL| as the base URL.
+          </li>
+          <li>If |url| is failure, return failure.
+          </li>
+          <li>If |url| is not [=manifest/within scope=] of |scope|, return
+          failure.
+          </li>
+          <li>Let |accept:ordered map| be an [=ordered map=].
+          </li>
+          <li>[=map/iterate=] |mime_type_string:string| → |extensions| of
+          |item|["accept"]
+            <ol>
+              <li>If |extensions| is not a [=string=] or [=list=], return
```

Comment:
I don't think we intend to support a string that's not in a list?

https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/manifest/manifest_parser.cc;l=1048-1057;drc=6fc55b5ee6ccec48d2e0ee47fb0e4bed0e1f8040

Reply:
Done